conf/vsc_calcua.config

```diff
@@ -31,6 +44,16 @@ if ( host == "antwerpen" ) {
             }
         }
     }
+    // APPTAINER_CACHEDIR/SINGULARITY_CACHEDIR
+    if ( !apptainer_cachedir ) {
+        System.err.println("\nWARNING: APPTAINER_CACHEDIR/SINGULARITY_CACHEDIR environment variable was not found.\nPlease add the line 'export APPTAINER_CACHEDIR=\"\${VSC_SCRATCH}/apptainer/cache\"' to your ~/.bashrc file (or set it with sbatch or in your job script).\nUsing the default storage location of Singularity/Apptainer ~/.apptainer/cache/. Read more about why this should be avoided in the VSC docs: https://docs.vscentrum.be/software/singularity.html#building-on-vsc-infrastructure\n")
```

Comment:
System.err vs warning, maybe should say 'ERROR' rather than warning? I would find that confusing (unless the `System.err` doesn't actually error/stop hte pipeline.

Reply:
Thanks for having a look!

System.err doesn't actually stop the pipeline - I believe it simply prints to stder instead of stdout - but this did confuse me as well coming back to the config after not looking at it for a while. 

The only places the config actually halts the process is where there's an explicit `System.exit(1)` statement (e.g., when mis-matching profiles/partitions), or a `try catch` (nxf cache dir not creatable,). I've tried to restrict these to bigger issues, and just warn the users for less severe sub-optimal usage (e.g., not setting a singularity cache).

I'll change those to ERROR instead of warning to make the distinction more clear.